Services/PortalMenuWidgetFilter.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Lombiq.HelpfulExtensions.Extensions.Widgets;
+using Lombiq.HelpfulExtensions.Extensions.Widgets.ViewModels;
+using Lombiq.HelpfulLibraries.Libraries.Mvc;
+using Lombiq.HelpfulLibraries.Libraries.Navigation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using OrchardCore.DisplayManagement;
+using OrchardCore.DisplayManagement.Layout;
+using OrchardCore.Navigation;
+using System.Threading.Tasks;
+using static Lombiq.BaseTheme.Constants.ZoneNames;
+
+namespace Lombiq.BaseTheme.Services
+{
+    public class PortalMenuWidgetFilter : WidgetFilterBase<PortalMenuWidgetViewModel>
```

Comment:
I think the word `Portal` is too specific. With this widget, you are providing the ability to render navigation _anywhere_. Maybe `NavigationWidgetFilter` would be more appropriate.

Reply:
I think here it's definitely justified becuase this filter is only for rendering "portal" navigation to the Navigation zone.

Reply:
Yeah, but it's in the base theme. It can be used for a menu for a blog or an agency website if the developer prefers it rather than CMS menus. I wouldn't call it "portal" then.

Reply:
So is the concern only about what type of site will use it as its main menu? Can we just call it "main" or "mainMenu" and call it a day?

Reply:
Sounds good.